conditional.tf

```diff
@@ -8,8 +8,7 @@ resource "template_dir" "flannel-manifests" {
   vars {
     flannel_image     = "${var.container_images["flannel"]}"
     flannel_cni_image = "${var.container_images["flannel_cni"]}"
-
-    pod_cidr = "${var.pod_cidr}"
+    pod_cidr          = "${var.pod_cidr}"
```

Comment:
Ditto formatting. How does it relate to kubernetes update?

Reply:
done